### PR TITLE
allow to use `-except` on post-processors

### LINKED ISSUE #2679: flag to skip postprocessor
I'd like a flag to skip certain postprocessors, for example atlas when I only want to do a local build.

### PR DESCRIPTION
**Pull request by azr**
This PR allows to use the already existing `--except` arg to skip a post-processor.

Notes:
*  `--only` ignores post-processors
* you can now name a pp & multiple pp can be named similarly !
*  existence of builders/pps passed to `--except` was/is not verified

Closes #2679

### CODE REVIEW

File: website/source/docs/commands/build.html.md [hunk 1]
```diff
@@ -44,9 +44,10 @@ artifacts that are created will be outputted at the end of the build.
     presents a prompt and waits for you to decide to clean up, abort, or retry
     the failed step.
 
--   `-only=foo,bar,baz` - Only build the builds with the given comma-separated
-    names. Build names by default are the names of their builders, unless a
-    specific `name` attribute is specified within the configuration.
+-   `-only=foo,bar,baz` - Only run the builds and post-processors with the
+    given comma-separated names. Build ans post-processor names by default are
```
Comment (rickard-von-essen) [inline website/source/docs/commands/build.html.md]: s/ans/and/

**Comment by rickard-von-essen** [comment]
Some thoughts about this:

1) This will change how `-only` works quite drastically. Before if you had builder A and B both with post-processor C `-only=A` resulted in `A->C`. After this only builder A will run without post-processing.

2) Builders can be named, but pp's doesn't have that option so you can't filter/run a specific pp definition if you have multiple of the same type.

3) What happens if you exclude a pp in the middle of a chain of pp's, will that be skipped or will the chain end there?

**Comment by azr** [comment]
Howdy ! 🙂 

1. Yes ! I thought of only allowing to `except` with pps but then this felt weird. May be this just means I should do another option ?
2. This PR also allows to name a pp
3. Good catch, for now it's a `continue`, in the loop so only that one is skipped. I will make it a `break` instead.

File: website/source/docs/commands/build.html.md (hunk 1 above)
Comment (azr) [reply website/source/docs/commands/build.html.md]: good catch, fixed !

**Comment by rickard-von-essen** [comment]
1. Don't know, the conservative way would be only allow `except` for pp's. 
2. 👍 
3. Both have their pros, but I think `break` is more intuitive and `continue` would in most cases just cause trouble.

**Comment by azr** [comment]
1. I totally agree, I made 'only' ignore pps
2. 👍 
3. Also agree, I pushed a commit to stop a chain of pp when one is skipped !

**Comment by azr** [comment]
Yaay, Thanks 😄 🎉

**Pull request merged**